superset/assets/src/views/chartList/ChartList.tsx

```diff
@@ -0,0 +1,316 @@
+    return this.hasPerm('can_edit');
+  }
+
+  get canDelete() {
+    return this.hasPerm('can_delete');
+  }
+
+  public static propTypes = {
+    addDangerToast: PropTypes.func.isRequired,
+  };
+
+  public state: State = {
+    chartCount: 0,
+    charts: [],
+    filterTypes: {},
+    labelColumns: {},
+    lastFetchDataConfig: null,
+    loading: false,
+    permissions: [],
+  };
+
+  public initialSort = [{ id: 'changed_on', desc: true }];
+
+  public columns = [
+    {
+      Cell: ({
+        row: {
+          original: { url, slice_name },
+        },
+      }: any) => <a href={url}>{slice_name}</a>,
+      Header: t('Chart'),
+      accessor: 'slice_name',
+      filterable: true,
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { viz_type },
+        },
+      }: any) => viz_type,
+      Header: t('Visualization Type'),
+      accessor: 'viz_type',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { datasource_name_text, datasource_link },
+        },
+      }: any) => <a href={datasource_link}>{datasource_name_text}</a>,
+      Header: t('Datasource'),
+      accessor: 'datasource_name_text',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_by_name, changed_by_url },
+        },
+      }: any) => <a href={changed_by_url}>{changed_by_name}</a>,
+      Header: t('Creator'),
+      accessor: 'creator',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_on },
+        },
+      }: any) => (
+          <span className='no-wrap'>{moment(changed_on).fromNow()}</span>
+        ),
+      Header: t('Last Modified'),
+      accessor: 'changed_on',
+      sortable: true,
+    },
+    {
+      Cell: ({ row: { state, original } }: any) => {
+        const handleDelete = () => this.handleChartDelete(original);
+        const handleEdit = () => this.handleChartEdit(original);
+        if (!this.canEdit && !this.canDelete) {
+          return null;
+        }
+
+        return (
+          <span className={`actions ${state && state.hover ? '' : 'invisible'}`}>
+            {this.canDelete && (
+              <ConfirmStatusChange
+                title={t('Please Confirm')}
+                description={<>{t('Are you sure you want to delete')} <b>{original.slice_name}</b>?</>}
+                onConfirm={handleDelete}
+              >
+                {(confirmDelete) => (
+                  <span
+                    role='button'
+                    className='action-button'
+                    onClick={confirmDelete}
+                  >
+                    <i className='fa fa-trash' />
+                  </span>
+                )}
+              </ConfirmStatusChange>
+            )}
+            {this.canEdit && (
+              <span
+                role='button'
+                className='action-button'
+                onClick={handleEdit}
```

Comment:
why not a link?

Reply:
I suppose it might make sense to do it this way, could bring in the edit modal from #9051 later and trigger a modal instead of following the url.

Reply:
That was my thinking. It's a link now, but will probably fire an action in the near future